src/cart/service/cart.service.ts

```diff
@@ -41,14 +50,15 @@ export class CartService {
 
         const productObjectId = new Types.ObjectId(productId);
```

Comment:
I can't see where you used this variable

Reply:
`    if (existingItem) {
                existingItem.quantity += quantity;
            } else {
                cart.items.push({ product: productObjectId, quantity });
            }`

Reply:
line 69